SPECS/gdb/gdb.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "gdb-11.2.tar.xz": "1497c36a71881b8671a9a84a0ee40faab788ca30d7ba19d8463c3cc787152e32"
+  "gdb-13.2.tar.xz": "fd5bebb7be1833abdb6e023c2f498a354498281df9d05523d8915babeb893f0a"
```

Comment:
I think the test pipelines fail: [link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=412045&view=logs&j=7909073b-e471-53c1-2c44-627e2efc0564&t=fbeef549-39ab-510c-faa3-78579a516b53&l=1230)
Did we upload the source tarball to the blobstore ?

Reply:
Hey @mfrw i don't have access to upload the sources.

Reply:
I've just uploaded the sources for this file

Reply:
I also ran another buddy build after the source upload. From a quick check, the package built for both architectures, but something errored out in the check tests. Can you please take a look?
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=412279&view=results

Reply:
So, i saw the tests failing on my end, it looks like the tests were failing before this change as well. I took a quick look, it looks like there are missing tcl scripts needed to run the test suit. I haven't had a chance to work out how to include them. As the tests were already failing before this change, do you still want me to focus on it?

Reply:
OK, pushed the fixes up to get the tests to work

Reply:
So it looks like gdb used to be apart of binutils, when it was pulled into its own package, the tcl scripts that define some of the test methods libctf uses were not brought across. I pulled them across and patched gdb to source them